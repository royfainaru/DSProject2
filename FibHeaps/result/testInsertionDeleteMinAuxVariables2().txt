╮
╰── ╭────────────────╮
    │ Key: 2         │
    ├────────────────┤
    │ Rank: 2        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: (null)   │
    │ Prev: (null)   │
    │ Child: 5       │
    ╰────────────────╯
    ├── ╭───────────────╮
    │   │ Key: 5        │
    │   ├───────────────┤
    │   │ Rank: 1       │
    │   │ Marked: false │
    │   │ Parent: 2     │
    │   │ Next: 3       │
    │   │ Prev: (null)  │
    │   │ Child: 6      │
    │   ╰───────────────╯
    │   ╰── ╭───────────────╮
    │       │ Key: 6        │
    │       ├───────────────┤
    │       │ Rank: 0       │
    │       │ Marked: false │
    │       │ Parent: 5     │
    │       │ Next: (null)  │
    │       │ Prev: (null)  │
    │       │ Child: (null) │
    │       ╰───────────────╯
    ╰── ╭───────────────╮
        │ Key: 3        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 2     │
        │ Next: (null)  │
        │ Prev: 5       │
        │ Child: (null) │
        ╰───────────────╯
