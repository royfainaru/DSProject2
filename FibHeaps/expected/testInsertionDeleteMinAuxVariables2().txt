╮
╰── ╭────────────────╮
    │ Key: 2         │
    ├────────────────┤
    │ Rank: 2        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: 2        │
    │ Prev: 2        │
    │ Child: 5       │
    ╰────────────────╯
    ├── ╭───────────────╮
    │   │ Key: 5        │
    │   ├───────────────┤
    │   │ Rank: 1       │
    │   │ Marked: false │
    │   │ Parent: 2     │
    │   │ Next: 3       │
    │   │ Prev: 3       │
    │   │ Child: 6      │
    │   ╰───────────────╯
    │   ╰── ╭───────────────╮
    │       │ Key: 6        │
    │       ├───────────────┤
    │       │ Rank: 0       │
    │       │ Marked: false │
    │       │ Parent: 5     │
    │       │ Next: 6       │
    │       │ Prev: 6       │
    │       │ Child: (null) │
    │       ╰───────────────╯
    ╰── ╭───────────────╮
        │ Key: 3        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 2     │
        │ Next: 5       │
        │ Prev: 5       │
        │ Child: (null) │
        ╰───────────────╯
